wiki/Mapping_Techniques/Inspiration_Gallery_Corner/en.md

```diff
@@ -0,0 +1,176 @@
+# Inspiration Gallery Corner
+
+> blame naraku
+
+Here is a gallery of miscellaneous and interesting sliders.
+
+## narakucrimson's Sliders of Wonderment
+
+![](img/narakucrimson-1.png)
+
+![](img/narakucrimson-2.png)
+
+![](img/narakucrimson-3.png)
```

Comment:
L13: Place a text inside ``[]`` to indicate which was supposed to be there if the image fails to load.